### ISSUE TITLE
The hMRI-toolbox: A toolbox for quantitative MRI and in vivo histology using MRI (hMRI).

### ISSUE THREAD

**Issue by ChristophePhillips** [ohbm/OpenScienceRoom2019#12]
**Title**

The hMRI-toolbox: A toolbox for quantitative MRI and in vivo histology using MRI (hMRI).

**Presentor and Affiliation**

- [Christophe Phillips](https://github.com/ChristophePhillips)

   [GIGA *in silico* medicine](https://www.gigainsilico.uliege.be/cms/c_4212667/en/portail-gigainsilico) & [GIGA Cyclotron Research Centre *in vivo* imaging](https://www.gigacrc.uliege.be/cms/c_4212477/en/portail-gigacrc). 
   [GIGA Institute](https://www.giga.uliege.be), University of Liège, Belgium.

- [Martina Callaghan](https://github.com/mfcallaghan)

   [Welcome Centre for Human Neuroimaging](https://www.fil.ion.ucl.ac.uk/), University College London, UK.

**Collaborators**

The development of the hMRI-toolbox is an international collaborative effort including the following sites and developers:

- Tobias Leutritz, Enrico Reimer, Nikolaus Weiskopf (Max Planck Institute for Human Cognitive and Brain Sciences, Leipzig, Germany)
- Evelyne Balteau, Christophe Phillips (University of Liege, Liege, Belgium)
- Siawoosh Mohammadi (Medical Center Hamburg-Eppendorf, Hamburg, Germany)
- Martina F Callaghan, John Ashburner (University College London, London, United Kingdom)
- Karsten Tabelow (Weierstrass Institute for Applied Analysis and Stochastics, Berlin, Germany)
- Bogdan Draganski, Ferath Kerif, Antoine Lutti (LREN, DNC - CHUV, University Lausanne, Lausanne, Switzerland)
- Maryam Seif (University of Zurich, Zurich, Switzerland)
- Gunther Helms (Department of Medical Radiation Physics, Lund University, Lund, Sweden)
- Lars Ruthotto (Emory University, Atlanta, GA, United States)
- Gabriel Ziegler (Otto-von-Guericke-University Magdeburg, Magdeburg, Germany)

**Github Link (if applicable)**

[`hMRI-toolbox`](https://github.com/hMRI-group/hMRI-toolbox)

**Abstract (max. 200 words):**

Neuroscience and clinical researchers are increasingly interested in quantitative magnetic resonance imaging (qMRI) due to its sensitivity to micro-structural properties of brain tissue such as axon, myelin, iron and water concentration ([Weiskopf et al., 2015](https://hmri-group.github.io/hMRI-toolbox/#Weiskopf.2015)).

The hMRI-toolbox is an easy-to-use open-source and flexible tool, for qMRI data handling and processing. It allows the estimation of high-quality multi-parameter qMRI maps (longitudinal and effective transverse relaxation rates R1 and R2*, proton density PD and magnetisation transfer MT saturation) ([Weiskopf et al., 2013](https://hmri-group.github.io/hMRI-toolbox/#Weiskopf.2013)), followed by spatial registration in common space for statistical analysis ([Draganski et al., 2011](https://hmri-group.github.io/hMRI-toolbox/#Draganski.2011)).

The qMRI maps generated by the toolbox can be used for quantitative parameter analysis and accurate delineation of subcortical brain structures. They are key input parameters for biophysical models designed to estimate tissue microstructure properties such as the MR g-ratio and to derive standard and novel MRI biomarkers ([Mohammadi et al., 2015](https://hmri-group.github.io/hMRI-toolbox/#Mohammadi.2015)). The hMRI toolbox is therefore the first step towards in vivo histology using MRI (hMRI) and is being extended further in this direction.

**Preferred Session**

`Demo: New advances in open neuroimaging methods`

**Additional Context**

Embedded in the [Statistical Parametric Mapping](http://www.fil.ion.ucl.ac.uk/spm) (SPM) framework, it can be readily combined with existing SPM toolboxes for estimating diffusion MRI parameter maps, and it benefits from the extensive range of established SPM tools for high-accuracy spatial registration and statistical inferences.

For a reference on the scientific background, methods and concepts please use this [NeuroImage paper](http://doi.org/10.1016/j.neuroimage.2019.01.029) and cite it when publishing results compiled with the hMRI-toolbox. A conference poster is also available [here](https://orbi.uliege.be/handle/2268/225763).

**Comment by TimVanMourik** [comment]
Hi @ChristophePhillips, I’m happy to tell you that we’d like to host your presentation as a lightning talk in the OSR in the **Collaborative research** session. This will be a talk of 5 minutes + 5 minutes of questions. We can unfortunately not offer you a slot in your preferred session as the ‘new advances’ session had many applicants yet limited availability. We hope this session is ok for you and would like to ask you if in your presentation you could briefly reflect on the session topic: the collaborative nature of your project.

We’ll update the program in the ReadMe.md shortly. We’d much appreciate it if you could submit slides and other presentation material to the [presentations folder](https://github.com/ohbm/OpenScienceRoom2019/tree/master/) by means of a Pull Request to this repository, preferably but not necessarily before the presentation.

**Comment by ChristophePhillips** [comment]
Hi @TimVanMourik ,
Thanks for the reply and I understand that the OSR is going to be very busy. I'll do my best to include the collaborative aspect of our hMRI toolbox. 
Regarding the date/time, it looks like the **Collaborative research** session is scheduled for Thursday 12:30-13:30, right ? Then I'd rather be presenting as early as possible since I'll have a poster to present on that day at about the same time... Thanks!

**Comment by TimVanMourik** [comment]
Thanks @ChristophePhillips for your understanding. I got another request for moving this session to a day earlier because of a timetable clash, so I would like to propose to swap the whole session to Wednesday. Might that also work better for you? Either way I'm happy to schedule as the first presentation.

**Comment by ChristophePhillips** [comment]
Hi @TimVanMourik .
Thanks for the flexibility. Unfortunately Wednesday will absolutely not be possible for me because of the Program Committee in-person meeting scheduled on Wednesday from 11am till ~2pm. On Tuesday I have the same contraints with a poster to present. Monday is my "lightest day" so far but it's not on your OSR agenda.
Good luck with the program !

**Comment by TimVanMourik** [comment]
Hi @ChristophePhillips, well, good, cause I messed up and the swap isn't possible for other reasons 😄
It sounds like it's best to put you early in the old slot, is that ok?

**Comment by ChristophePhillips** [comment]
Yes, all fine. Let's go for Thursday, at the beginning of the slot ! :-)
